src/pip_sync_faster/__main__.py

```diff
@@ -0,0 +1,5 @@
+import sys
+
+from pip_sync_faster.cli import cli
+
+sys.exit(cli())
```

Comment:
It'll be a good idea to move this `__main__.py` into the cookiecutter where it can be implemented correctly once and for all as it's actually not as trivial as you might expect.

`cli()` is used as both the setuptools entry point function (specified in `setup.cfg`) and the top-level function `__main__.py` to call. A setuptools entry point function is expected to return something appropriate for being passed to [`sys.exit()`](https://docs.python.org/3/library/sys.html#sys.exit): `None` or `0` to exit successfully, an int to exit with that error code, or any printable object (such as a string) that will be printed to stderr before exiting with `1`. You can see this by looking at the `pip-sync-faster` script that setuptools generates when you install the package:

```terminal
$ cat .tox/dev/bin/pip-sync-faster
#!/home/seanh/Projects/pip-sync-faster/.tox/dev/bin/python
# -*- coding: utf-8 -*-
import re
import sys
from pip_sync_faster.cli import cli
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(cli())
```

A `__main__.py` or an `if __name__ == "__main__"` block should do the same thing: `sys.exit(cli())`.

This means that `cli()` itself shouldn't call `sys.exit()`, it should just return `None` or `0` to exit successfully or `return 1` or `return "Some error message"` to exit with an error

Reply:
TIL about sys.exit with a function, nice.